projects/docs-tools/src/modules/type-definitions/property-definitions.component.scss

```diff
@@ -10,7 +10,15 @@
 .sky-docs-property-definitions {
   width: 100%;
   max-width: 100%;
-  overflow: auto;
```

Comment:
This caused sticky headers to not stick. I removed it and compensated for its behavior by adding `word-break: break-word` to the description table cell in the off chance that it contains a word that's long enough to cause the table to expand past the page width.